### PR TITLE
Xuanying #957 disable carousel auto advance

### PR DESCRIPTION
**Pull request by SleeplessRegina**
Disable carousel auto advance, make the carousel only support manually advance both in TestimonialCarousel and BlogCarousel.
Set the custom intervals to null and remove interval from our type statement.

### CODE REVIEW

File: common/components/controllers/CorporateHackathonController.jsx [hunk 1]
```diff
@@ -263,7 +263,7 @@ class CorporateHackathonController extends React.PureComponent<{||}, State> {
           <div className="corporate-hackathon-carousel carousel-testimonial-root">
             <TestimonialCarousel
               category="partner-highlights"
-              interval={600000}
+              interval={null}
```
Comment (PeterBreen) [inline common/components/controllers/CorporateHackathonController.jsx:266]: Do we need to use interval={null} here, as well as in the TestimonialCarousel component itself? Carousel shouldn't be taking any props so I'm hoping it's safe to remove, but please test and let me know?

File: common/components/controllers/CorporateHackathonController.jsx [hunk 2]
```diff
@@ -409,7 +409,7 @@ class CorporateHackathonController extends React.PureComponent<{||}, State> {
           <div className="corporate-hackathon-carousel carousel-testimonial-root">
             <TestimonialCarousel
               category="hackathon-highlights"
-              interval={600000}
+              interval={null}
```
Comment (PeterBreen) [inline common/components/controllers/CorporateHackathonController.jsx:412]: Same question as above

File: common/components/controllers/LandingController.jsx [hunk 3]
```diff
@@ -363,7 +363,7 @@ class LandingController extends React.PureComponent<{||}, State> {
         <h2 className="text-center LandingController-section-header">
           What people are saying about DemocracyLab
         </h2>
-        <TestimonialCarousel interval={15000} />
+        <TestimonialCarousel interval={null} />
```
Comment (PeterBreen) [inline common/components/controllers/LandingController.jsx:366]: Same question as above

**Review by PeterBreen — CHANGES_REQUESTED** [review]
It looks good, but please see the three line comments on instances of using interval as a prop that we may not need. But please do let me know if it is necessary :)

**Comment by SleeplessRegina** [comment]
> It looks good, but please see the three line comments on instances of using interval as a prop that we may not need. But please do let me know if it is necessary :)

Yes, I removed the interval props of TestimonialCarousel then tested again and found it still looks good. Both the carousel won't automatically advance.

**Review by PeterBreen — APPROVED** [review]
Thanks for those changes; looks good, works exactly as expected 👍

**Pull request merged**